cmd/kueuectl/app/list/list_workload_test.go

```diff
@@ -0,0 +1,471 @@
+				},
+			},
+			objs: []runtime.Object{
+				utiltesting.MakeWorkload("wl1", metav1.NamespaceDefault).
+					OwnerReference(batchv1.SchemeGroupVersion.WithKind("Job"), "j1", "test-uid").
+					Queue("lq1").
+					Active(true).
+					Admission(utiltesting.MakeAdmission("cq1").Obj()).
+					Condition(metav1.Condition{
+						Type:   kueue.WorkloadQuotaReserved,
+						Status: metav1.ConditionFalse,
+						Reason: "Pending",
+					}).
+					Creation(testStartTime.Add(-1 * time.Hour).Truncate(time.Second)).
+					Obj(),
+				utiltesting.MakeWorkload("wl2", metav1.NamespaceDefault).
+					OwnerReference(rayv1.GroupVersion.WithKind("RayJob"), "j2", "test-uid").
+					Queue("lq2").
+					Active(true).
+					Admission(utiltesting.MakeAdmission("cq2").Obj()).
+					Condition(metav1.Condition{
+						Type:   kueue.WorkloadQuotaReserved,
+						Status: metav1.ConditionFalse,
+						Reason: "Pending",
+					}).
+					Creation(testStartTime.Add(-2 * time.Hour).Truncate(time.Second)).
+					Obj(),
+			},
+			wantOut: `NAME   JOB TYPE   JOB NAME   LOCALQUEUE   CLUSTERQUEUE   STATUS    POSITION IN QUEUE   AGE
+wl1               j1         lq1          cq1            PENDING   12                  60m
+wl2               j2         lq2          cq2            PENDING   22                  120m
+`,
+		},
+		"should print not found error": {
+			wantOutErr: fmt.Sprintf("No resources found in %s namespace.\n", metav1.NamespaceDefault),
+		},
+		"should print not found error with all-namespaces filter": {
+			args:       []string{"-A"},
+			wantOutErr: "No resources found\n",
+		},
+	}
+	for name, tc := range testCases {
+		t.Run(name, func(t *testing.T) {
+			streams, _, out, outErr := genericiooptions.NewTestIOStreams()
+
+			tf := cmdtesting.NewTestClientGetter()
+			if len(tc.ns) > 0 {
+				tf.WithNamespace(tc.ns)
+			}
+
+			clientset := fake.NewSimpleClientset(tc.objs...)
+
+			// `SimpleClientset` not allow to add `PendingWorkloadsSummary` objects,
+			// because of `PendingWorkload` resources not implement `runtime.Object`.
+			// Default `Reaction` handle all verbs and resources, so need to add on
+			// head of chain.
+			clientset.Fake.ReactionChain = append([]kubetesting.Reactor{
+				&kubetesting.SimpleReactor{
+					Verb:     "get",
+					Resource: "clusterqueues",
+					Reaction: func(action kubetesting.Action) (handled bool, ret runtime.Object, err error) {
+						obj := &v1alpha1.PendingWorkloadsSummary{Items: tc.pendingWorkloads}
+						return true, obj, err
+					},
+				},
+			}, clientset.Fake.ReactionChain...)
+
+			tf.ClientSet = clientset
+			tf.ClientSet.Discovery().(*fakediscovery.FakeDiscovery).Resources = tc.apiResourceLists
+
+			cmd := NewWorkloadCmd(tf, streams)
+			cmd.SetArgs(tc.args)
+
+			gotErr := cmd.Execute()
+			if diff := cmp.Diff(tc.wantErr, gotErr, cmpopts.EquateErrors()); diff != "" {
+				t.Errorf("Unexpected error (-want/+got)\n%s", diff)
+			}
+
+			gotOut := out.String()
+			fmt.Println(gotOut)
```

Comment:
```suggestion
```

Reply:
Or you can use `t.Log`, if really necessary.

Reply:
Removed